src/execution/vm/vm.cpp

```diff
@@ -459,7 +459,7 @@ void VM::Interpret(const uint8_t *ip, Frame *frame) {  // NOLINT
 
   OP(ExecutionContextAddRowsAffected) : {
     auto *exec_ctx = frame->LocalAt<exec::ExecutionContext *>(READ_LOCAL_ID());
-    auto rows_affected = frame->LocalAt<int64_t>(READ_LOCAL_ID());
+    auto rows_affected = frame->LocalAt<int32_t>(READ_LOCAL_ID());
```

Comment:
Patch suggested by @lmwnshn! Might be worth reverting this part once #1049 gets resolved.